circuit/circuit.lp

```diff
@@ -356,6 +393,15 @@
   (name "GND3") (value "GND")
   (signal ff161c97-29a5-43aa-a9ae-3ca7a66982ce (net 7bce9215-22e4-4be8-9bec-7495ce622913))
  )
+ (component 6c1d9680-4b28-4ba5-b045-0dca29323524
+  (lib_component f7b52b62-7814-4a74-88ac-155e101507d5)
+  (lib_variant 75a3b2bb-300f-4ef0-9f8e-dbe84c4c4d55)
+  (lib_device f7139d7c-a7df-4cb2-af92-06b38f7a12f4)
+  (name "J7") (value "{{PARTNUMBER}}")
```

Comment:
Can we add a comment to the value about that we use it as a selection jumper? Since we can't freely add text labels.

Reply:
Done!